### PR TITLE
Loosen requests version.

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
     license='MIT',
     packages=setuptools.find_packages(exclude=['tests', 'tests.*']),
     install_requires=[
-        'requests == 2.27.1'
+        'requests >= 2.27.1'
```
Comment (Natim) [inline setup.py:23]:
```suggestion
        'requests'
```

**Comment by Armando-Ibarra** [comment]
Sound good!

**Comment by Natim** [comment]
Thank you @Armando-Ibarra .
Do you know when the next release is planned?

**Comment by Armando-Ibarra** [comment]
@Natim I can probably release it tomorrow, really we don't have any other updates addressed for now.
Is this something urgent for you?

**Comment by Natim** [comment]
Thank you @Armando-Ibarra I will do the update and keep you posted.

**Pull request merged**